__tests__/test.js

```diff
@@ -0,0 +1,37 @@
+const request = require('supertest')
+const db = require('../db/connection')
+const app = require('../app')
+const seed = require('../db/seeds/seed')
+const seedTestData = require('../db/data/test-data')
+
+beforeEach(() => {
+    return seed(seedTestData);
+  });
+
+afterAll(() => {
+    db.end();
+});
+
+describe('invalid url', () => {
+    test('GET request with an invalid endpoint, should return 404 and a message', () => {
+        return request(app).get('/api/notgonnawork').expect(404).then(({error}) => {
+            expect(`${error}`).toBe('Error: cannot GET /api/notgonnawork (404)')
+        })
+    })
+})
+
+describe('/api/topics', () => {
+    test('GET request should respond 200 and respond with an array of all the topic objects', () => {
+        return request(app).get('/api/topics').expect(200).then(({body}) => {
+            expect(Array.isArray(body)).toBe(true)
+        })
+    })
+    test('GET request should respond with array containing objects with slug and description properties', () => {
+        return request(app).get('/api/topics').expect(200).then(({body}) => {
+            body.forEach((object) => {
+                expect(object).toHaveProperty('slug')
+                expect(object).toHaveProperty('description')
```

Comment:
You have already asserted for a `200` status code in the first test, so you don't also need to have it in the second test to save code

You could be more robust here by checking the `typeof` data each value would be instead